src/current/releases/cloud.md

```diff
@@ -188,7 +186,7 @@ Get future release notes emailed to you:
 
 <h3 id="2024-01-17-general-updates"> General updates </h3>
 
-- The single-page CockroachDB Cloud [Create cluster]({% link cockroachcloud/create-a-serverless-cluster.md %}) and [Edit cluster]({% link cockroachcloud/basic-cluster-management.md %}) have been updated to use multi-step wizards.
+- The single-page CockroachDB Cloud [Create cluster]({% link cockroachcloud/create-a-basic-cluster.md %}) and [Edit cluster]({% link cockroachcloud/basic-cluster-management.md %}) have been updated to use multi-step wizards.
```

Comment:
Should "Cloud" be using the liquid var?

Reply:
We don't require it really, it doesn't provide much benefit. We actually went away from it for a while and then seem to have begun using it again. I'm neutral on it and will leave it alone here.